mcs/class/System/common_networking.sources

```diff
@@ -20,12 +20,17 @@ System.Net/AuthenticationManager.cs
 System.Net/ChunkedInputStream.cs
 System.Net/EndPointListener.cs
 System.Net/EndPointManager.cs
-System.Net/FtpAsyncResult.cs
-System.Net/FtpDataStream.cs
+
 System.Net/FtpRequestCreator.cs
```

Comment:
This can be replaced by FtpWebRequestCreator class in corefx/src/System.Net.Requests/src/System/Net/FtpWebRequest.cs

Reply:
@akoeplinger thanks! However this change will require changes in machine.configs

Reply:
That can be done with follow up commit